web/crux/src/app/image/image.mapper.ts

```diff
@@ -143,11 +146,14 @@ export default class ImageMapper {
   }
 
   configSectionResetToDb(config: Partial<ContainerConfigData>, section: string) {
-    if (CONTAINER_CONFIG_JSON_FIELDS.includes(section)) {
-      config[section] = Prisma.JsonNull
-    } else {
-      config[section] = null
-    }
+    const fields = CONTAINER_CONFIG_COMPOSITE_FIELDS[section] ?? [section]
```

Comment:
What's the logic behind all this?

Reply:
If the user resets the 'storage' config on the UI, in the DB it should reset multipe fields.